flink-connector-mysql-cdc/src/main/java/com/alibaba/ververica/cdc/connectors/mysql/table/MySQLTableSourceFactory.java

```diff
@@ -75,6 +76,11 @@
 			"MySQL database cluster as another server (with this unique ID) so it can read the binlog. " +
 			"By default, a random number is generated between 5400 and 6400, though we recommend setting an explicit value.");
 
+	private static final ConfigOption<Map<String, String>> PROPERTIES = ConfigOptions.key("properties")
+		.mapType()
+		.noDefaultValue()
+		.withDescription("The Debezium MySQL connector properties.");
+
```

Comment:
Currently, `ConfigOption` doesn't support Map type, and compress all the properties in one entry is not user-friendly. Could you take the implementation of Kafka `properties.*` as an example? see https://github.com/apache/flink/blob/master/flink-connectors/flink-connector-kafka-base/src/main/java/org/apache/flink/streaming/connectors/kafka/table/KafkaOptions.java#L292

Reply:
OK, i will modify it, and add the support for PG CDC .